src/main/java/io/cdap/e2e/pages/locators/CdfSysAdminLocators.java

```diff
@@ -51,4 +53,92 @@ public class CdfSysAdminLocators {
 
   @FindBy(how = How.XPATH, using = "//*[@data-cy='Clear All']")
   public static WebElement clearAll;
+  @FindBy(how = How.XPATH, using = "//span//button[contains(@class, 'btn btn-secondary')]")
+  public static WebElement reloadSystemArtifacts;
+  @FindBy(how = How.XPATH, using = "//a[@href='/cdap/httpexecutor']")
+  public static WebElement makeHttpCall;
+  @FindBy(how = How.XPATH, using = "//button[@data-testid='create-namespace-btn']")
+  public static WebElement createNewNamespace;
+  @FindBy(how = How.XPATH, using = "//button[@data-testid='edit-system-prefs-btn']")
+  public static WebElement editSystemPreference;
+  @FindBy(how = How.XPATH, using = "//a[@href='/cdap/ns/system/profiles/create']")
+  public static WebElement createComputeProfile;
+  @FindBy(how = How.XPATH, using = "//div//label[@for='import-profile']")
+  public static WebElement clickImport;
+  @FindBy(how = How.XPATH, using = "//button[@data-testid='save-prefs-btn']")
+  public static WebElement clickSaveClose;
+  @FindBy(how = How.XPATH, using = "//input[@placeholder='Namespace name' and @type='text']")
```

Comment:
We have a data-test-id for this we can use it and create something like this :

```//div[@data-testid='create-namespace-name']//input```

Reply:
Change done.